internal/backend/oauth/oauth.go

```diff
@@ -701,6 +719,8 @@ func (o *oauth) Exchange(ctx context.Context, integration string, cid sdktypes.C
 		return nil, fmt.Errorf("bad oauth integration name: %w", err)
 	}
 
+	o.logger.Warn("Exchange", zap.String("code", code))
+
```

Comment:
This was needed to debug the scopes issue, but if it remains here it'll leak user secrets
```suggestion
```